core/jobs.py

```diff
@@ -1429,11 +1429,16 @@ def _clear_inactive_realtime_layer(cls, latest_created_on_datetime):
 
     @classmethod
     def _kickoff_batch_job(cls):
```

Comment:
Maybe rename to _kickoff_batch_job_and_return_job_id?

Reply:
I don't think it's necessary, other method names don't follow that convention and I think receiving a job_id from a "start job" function is already intuitive enough.